pkg/operator/starter.go

```diff
@@ -44,7 +44,10 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
+	dynamicClientConfig := dynamic.ConfigFor(ctx.KubeConfig)
+	dynamicClientConfig.Burst = 40
```

Comment:
we need to experiment and find some good numbers for QPS/Burst

Reply:
What does kube-storage-version-migrator use?

Reply:
Please rename `dynamicClient` to `migrationClient` to make clear it is not meant for anything else than migration.

Reply:
> What does kube-storage-version-migrator use?

it seems like they use default settings https://github.com/kubernetes-sigs/kube-storage-version-migrator/blob/master/cmd/migrator/app/migrator.go#L58